protocols/grpc/health.pb.go

```diff
@@ -108,7 +110,10 @@ func init() {
 }
 func (this *CheckRequest) Equal(that interface{}) bool {
 	if that == nil {
-		return this == nil
+		if this == nil {
```

Comment:
The check is useless yet it is auto-generated. Are you using an old version of protoc?

Reply:
I am try to using the latest `protoc` to re-generated it :-)